app/templates/500.html

```diff
@@ -9,10 +9,7 @@
         <div class="govuk-grid-column-two-thirds">
             <h1 class="govuk-heading-xl">Sorry, there is a problem with the service</h1>
             <p class="govuk-body">Try again later.</p>
-            <p class="govuk-body">We have not saved your answers. When the service is available, you will have to start
-                again. </p>
-            <p class="govuk-body">Contact the Community Ownership Fund if you have any questions. </p>
-            {{ contact_details(contact_us_email_address, contact_us_phone, contact_us_text_phone, opening_days, opening_time, closed) }}
+            <p class="govuk-body"><a class="govuk-link" href="{{ contact_us_url }}">Contact the Community Ownership Fund</a> if you have any questions.</p>
```

Comment:
Where is `contact_us_url` being populated?

Reply:
Done